news/pymoab_cleanup.rst

```diff
@@ -0,0 +1,54 @@
+**Added:** None
+
+* Testing PyNE build without PyMOAB using Python 2 & 3
+* Added FindDAGMC.cmake file
+
+**Changed:** 
+
+* PyNE can be build with PyMOAB whitout DAGMC with limited capabilities
+
+* "--dagmc" flag added to `setup.py` in order to build PyNE against DAGMC
+
+* Add Test to check PyNE build with PyMOAB (and DAGMC) using Python 2
+
+* pyne.mesh now takes advantage of PyMOAB instead of PyTAPS:
+   - IMeshTag -> NativeMeshTag
+   - 'imesh' -> 'nat_mesh'
+   - mesh.destroyTag(self, boolean) -> mesh.delete_tag(self)
+   - mesh.tag.type -> mesh.tag.get_dtype()
+   - write_hdf5(self, filename) -> write_hdf5(self, filename, write_mats)
+   - imesh.iterate() -> mesh.mesh_iterate()
+   - iMesh.Mesh().load(hdf5) -> core.Core().load_file(hdf5)
+   - iMesh.Mesh().getEntSets() -> core.Core().get_entities_by_type(0, types.MBENTITYSET)
+   - iMesh.Mesh().getTagHandle('CATEGORY') -> core.Core().get_entities_by_type(0, types.MBENTITYSET)
+   - iMesh.Mesh().getTagHandle('GLOBAL_ID') -> core.Core().tag_get_handle(types.GLOBAL_ID_TAG_NAME)
+   - iMesh.Mesh().getTagHandle('NAME') -> core.Core().tag_get_handle(types.NAME_TAG_NAME)
+   - iMesh.mesh.getTagHandle('cell_number') -> mesh.cell_number
```

Comment:
These changes are not due to pyMOAB. This was already available in IMeshTag but was rarely used.

When the tag is created on a mesh object, it is added to the attribute list of that mesh object.